packages/@aws-cdk/aws-eks/test/sdk-call-integ-test-docker-app/app/sdk-call.js

```diff
@@ -0,0 +1,11 @@
+const sdk = require('aws-sdk');
```

Comment:
We can avoid having to create a docker image by using config maps to mount code onto containers. 

See https://cdk8s.io/docs/latest/plus/cdk8s-plus-23/config-map/#creating-a-volume-from-a-directory

This will reduce the complexity of the test, and remove the dependency on ECR.